lib/ansiblelint/__init__.py

```diff
@@ -138,6 +139,15 @@ def matchyaml(self, file, text):
         return matches
 
 
+class LoadingFailureRule(AnsibleLintRule):
```

Comment:
And `AnsibleLintRule` could go into `rules/__init__.py` to avoid circular imports.